cibuild.cmd

```diff
@@ -1,17 +1,31 @@
+@setlocal enabledelayedexpansion
+
+REM Parse Arguments.
+
+set RoslynRoot=%~dp0
+set BuildConfiguration=Debug
+:ParseArguments
+if "%1" == "" goto :DoneParsing
+if /I "%1" == "/?" call :Usage && exit /b 1
+if /I "%1" == "/debug" set BuildConfiguration=Debug&&shift&& goto :ParseArguments
+if /I "%1" == "/release" set BuildConfiguration=Release&&shift&& goto :ParseArguments
+call :Usage && exit /b 1
```

Comment:
This was changed from "goto" to "call" to ensure the "exit /b 1" executes properly.